Fix 3817 map info url (#4000)
geosolutions-it/MapStore2: docs/developer-guide/map-query-parameters.md

> # Map query parameters

Reviewer: ```suggestion
# MapViewer query parameters
```

> In this section we will describe the available map query parameters

Reviewer: I would name these parameters MapViewer parameters, but this is just my opinion.

that can be used when the map is loaded.

## Retro-compatibility

> In the past it has been introduced a **bbox** query param with value that corresponds to extent in the viewport.

Reviewer: ```suggestion
In the past a **bbox** query param has been introduced whose value corresponds to extent in the viewport.
```

for example:
`?bbox=-177.84667968750014,-1.8234225930143395,-9.096679687500114,61.700290838326204`

For more details on it see [sharing a map](../user-guide/share).

> ## New way to dispatch actions in MapStore

Reviewer: ```suggestion
## Dynamically dispatching initial actions in MapStore
```

> **actions** is the new query param and it will be used to dispatch actions (in the map viewer) that are limited to a whitelist.

Reviewer: ```suggestion
to dispatch additional actions when the map viewer is started, the **actions**  query param can be used. Only actions from a configured whitelist can be dispatched in this way.
```

#### Map info
It performs a [GetFeature](https://docs.geoserver.org/stable/en/user/services/wfs/reference.html#getfeature) request and then a [GetFeatureInfo](https://docs.geoserver.org/stable/en/user/services/wms/reference.html#getfeatureinfo) by taking a geometry from a feature retrieved. This action can be used can be used only for existing maps.

With the GetFeature request it takes the first coordinate of the geometry of the first feature.
Then it uses it for the GFI request by creating an area of 101x101 px and by limiting it to the specified layer.

You can add to the action a cql_filter that will be used in both request.

Requirements:
- the layer specified must be visible in the map
- there must be a geometry that can be retrieved from the GetFeature request

Example:
```
{
    "type": "SEARCH:SEARCH_WITH_FILTER",
    "cql_filter": "ID=75",
    "layer": "WORKSPACE:LAYER_NAME"
}
?actions=[{"type":"SEARCH:SEARCH_WITH_FILTER","cql_filter":"ID=75","layer":"WORKSPACE:LAYER_NAME"}]
```
For more details check out the [searchLayerWithFilter](https://mapstore2.geo-solutions.it/mapstore/docs/#actions.search.exports.searchLayerWithFilter) in the framework documentation

> #### Add Layers (not implemented yet)

Reviewer: I would not document something that is not implemented yet. You can add this documentation together with the related implementation.
